pkg/apis/core/types.go

```diff
@@ -3718,6 +3718,20 @@ type Node struct {
 	Status NodeStatus
 }
 
+// We define LabelNodeRoleMaster and LabelNodeRoleNode, which are the preferred
+// form, but we will also recognize other forms that are known to be in
+// widespread use (NodeLabelKubeadmAlphaRole, NodeLabelRole, etc) .
+const (
+	// LabelNodeMasterRole is set to the value "true" indicating a master node.
+	// A master node typically runs kubernetes system components and will not typically run user workloads.
+	// When set to any value other than "true" it is ignored.
+	LabelNodeRoleMaster = "node-role.kubernetes.io/master"
```

Comment:
BTW "master" should be changed to "control-plane". this is a project wide decision.

kubeadm has started moving away from "master", yet for the label value itself it's going to be trickier.
for now we can at least name the constant - LabelNodeRoleControlPlane.

Reply:
Agree with that. The name `LabelNodeRoleMaster`  also makes me confused.